tests/cypress/tests/standalone/tooltip_test.ts

```diff
@@ -29,5 +29,72 @@ describe(`tooltips`, () => {
         cy.wait(5000);
         cy.get('.dash-table-tooltip').should('not.be.visible');
     });
+});
 
+describe(`tooltips, mode=${AppMode.FixedTooltips}`, () => {
+    beforeEach(() => {
+        cy.visit(`http://localhost:8080?mode=${AppMode.FixedTooltips}`);
+    });
+
+    it('displays in fixed column', () => {
+        DashTable.getCellById(6, 'rows').trigger('mouseover');
+        cy.get('.dash-table-tooltip').should('not.be.visible');
+        cy.wait(0);
+        cy.get('.dash-table-tooltip').should('be.visible');
+        cy.get('.dash-table-tooltip').then(t => {
+            expect(t[0].innerText).to.equal('Learn the eyestealing tesuji');
+            expect(!!t[0].children && !!t[0].children.length).to.equal(false);
+
+            const bounds = t[0].getBoundingClientRect();
+
+            DashTable.getCellById(6, 'rows').then(table => {
+                const cellBounds = table[0].getBoundingClientRect();
+
+                expect(bounds.top).to.be.greaterThan(cellBounds.top + cellBounds.height);
+            });
+        });
+        cy.wait(5000);
+        cy.get('.dash-table-tooltip').should('not.be.visible');
+    });
+
+    it('displays in fixed row', () => {
+        DashTable.getCellById(0, 'ddd').trigger('mouseover');
+        cy.get('.dash-table-tooltip').should('not.be.visible');
+        cy.wait(0);
+        cy.get('.dash-table-tooltip').should('be.visible');
+        cy.get('.dash-table-tooltip').then(t => {
+            expect(t[0].innerText).to.equal('Hane, Cut, Placement');
+
+            const bounds = t[0].getBoundingClientRect();
+
+            DashTable.getCellById(0, 'ddd').then(table => {
+                const cellBounds = table[0].getBoundingClientRect();
+
+                expect(bounds.top).to.be.greaterThan(cellBounds.top + cellBounds.height);
+            });
+        });
+        cy.wait(5000);
+        cy.get('.dash-table-tooltip').should('not.be.visible');
+    });
+
+    it('displays in fixed row & column', () => {
+        DashTable.getCellById(0, 'rows').trigger('mouseover');
+        cy.get('.dash-table-tooltip').should('not.be.visible');
+        cy.wait(0);
+        cy.get('.dash-table-tooltip').should('be.visible');
+        cy.get('.dash-table-tooltip').then(t => {
+            expect(t[0].innerText).to.equal('Learn the eyestealing tesuji');
+            expect(!!t[0].children && !!t[0].children.length).to.equal(false);
+
+            const bounds = t[0].getBoundingClientRect();
+
+            DashTable.getCellById(0, 'rows').then(table => {
+                const cellBounds = table[0].getBoundingClientRect();
+
+                expect(bounds.top).to.be.greaterThan(cellBounds.top + cellBounds.height);
+            });
```

Comment:
This test is a bit crude and tests that the tooltip is placed below the cell. If the adjustment is not run the tooltip is in its default position and will be above the cell (top=0, left=0)